KidsIdKit/KidsIdKit.csproj.bak

```diff
@@ -0,0 +1,128 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Razor">
```

Comment:
The .bak file shouldn't get pushed into source control

Did I forget to put a .gitignore file in this repo?

Reply:
I see that there is a .gitignore, but it doesn't have a `*.csproj.bak` entry.

Now I'm curious. What dev tool generated the bak file?